misc/testerina/modules/testerina-core/src/main/ballerina/src/test/annotations.bal

```diff
@@ -33,17 +33,13 @@ public type TestConfig record {
 
 # Configuration of the function to be mocked.
 #
-# + moduleName - Name of the module that the function to be mocked resides in
 # + functionName - Name of the function to be mocked
 public type MockConfig record {
-    string moduleName = ".";
     string functionName = "";
```

Comment:
how do we support mocking other packages ?
@test:Mock { functionName: "io:println" }

Reply:
Added previous mock function implementation temporarily to address this issue. Pushed changes.